spec/views/flags/show.html.erb_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe "flags/show", type: :view do
+  before(:each) do
+    @flag = assign(:flag, Flag.create!(
+      :name => "Name",
```

Comment:
Indent the first parameter one step more than `Flag.create!(`.<br>Use the new Ruby 1.9 hash syntax.